actix-web-codegen/src/lib.rs

```diff
@@ -196,6 +196,7 @@ method_macro!(Connect, connect);
 method_macro!(Options, options);
 method_macro!(Trace, trace);
 method_macro!(Patch, patch);
+method_macro!(All, all);
```

Comment:
This will almost certainly be easier with a new macro creator rather than trying to retro-fit `method_macro!`

Reply:
Is making a custom http server and defining any a way or is there a better way? @robjtede

Reply:
What it would need to generate would be similar to the other macros (an HttpServiceFactory implementation) except without adding any method guards to the route.

Reply:
I am sorry but I am not sure if I understand @robjtede

Reply:
![Screenshot 2024-03-17 123942](https://github.com/actix/actix-web/assets/123313687/6faea30b-693a-400d-8b06-249c1067d54e)
Do I need to define it here?
![Screenshot 2024-03-17 124134](https://github.com/actix/actix-web/assets/123313687/346d99d4-1e01-42fd-bf7c-83b2eb8fac0a)
Or in service.rs in actix-http. 
Pardon me for asking so many questions btw